website/source/docs/platform/k8s/helm.html.md

```diff
@@ -230,6 +232,19 @@ and consider if they're appropriate for your deployment.
             annotations: |
               "annotation-key": "annotation-value"
             ```
+* <a name="v-externalservers" href="#v-externalservers">`externalServers`</a> - Configuration for Consul servers running externally. This information is required if Consul servers are running outside of k8s and you’re setting `global.tls.enableAutoEncrypt` to `true`.
+
+  * <a name="v-externalservers-enabled" href="#v-externalservers-enabled">`enabled`</a> (`boolean: false`) - If true, the chart will talk to external servers configured here.
+
+  * <a name="v-externalservers-https" href="#v-externalservers-https">`https`</a> - HTTPS configuration for external servers. Note: HTTP connections to the servers are not supported.
+
+      * <a name="v-externalservers-address" href="#v-externalservers-address">`address`</a> (`string: null`) - IP, DNS name, or Cloud auto-join string pointing to the external Consul servers. Note that if you’re providing the cloud auto-join string and multiple addresses can be returned, only the first address will be used. This value is required only if you would like to use a different server address from the one specified in the `client.join` property.
```

Comment:
Is Cloud auto-join supposed to be capitalized? Can you link to https://www.consul.io/docs/agent/cloud-auto-join.html as well, i.e. [cloud auto-join](https://www.consul.io/docs/agent/cloud-auto-join.html).

Reply:
Yep, you're right, it doesn't need to be capitalized. Added the link too. Thanks!